openshift/keycloak.app.yaml

```diff
@@ -181,3 +181,7 @@ objects:
 parameters:
 - name: IMAGE_TAG
   value: latest
+- name: OPERATING_MODE
+  value: clustered
+- name: REPLICAS
+  value: 3
```

Comment:
This has to be string otherwise `oc process` will complain

```
error: unable to decode "keycloak-templates/keycloak-deployment.yaml": [pos 3892]: json: expect char '"' but got char '3'
```